pkgs/applications/science/medicine/xmedcon/default.nix

```diff
@@ -24,11 +26,15 @@ stdenv.mkDerivation rec {
     zlib
   ];
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config wrapGAppsHook ];
+
+  shellHook = ''
```

Comment:
I don't think we set shellHook anywhere in nixpkgs, and I don't think we patch in random icon themes either. These are usually picked up ambiently from the environment, so I'd skip that part.